addons/web/static/tests/debug/debug_manager_tests.js

```diff
@@ -185,4 +188,37 @@ QUnit.module("DebugManager", (hooks) => {
       target.querySelector(".o_dialog_container").remove();
     }
   );
+
+  QUnit.test("can regenerate assets bundles", async (assert) => {
+    const mockRPC = async (route, args) => {
+      if (args.method === "check_access_rights") {
+        return Promise.resolve(true);
+      }
+      if (route === "/web/dataset/call_kw/ir.attachment/search") {
+        assert.step("ir.attachment/search");
+        return [1, 2, 3];
+      }
+      if (route === "/web/dataset/call_kw/ir.attachment/unlink") {
+        assert.step("ir.attachment/unlink");
+        return Promise.resolve(true);
+      }
+    };
+    testConfig = { mockRPC };
+    patchWithCleanup(browser, {
+      location: {
+        reload: () => assert.step("reloadPage"),
+      },
+    });
+    serviceRegistry.add("localization", makeFakeLocalizationService());
+    debugRegistry.add("regenerateAssets", regenerateAssets);
+    const env = await makeTestEnv(testConfig);
+    const debugManager = await mount(DebugManager, { env, target });
+    await click(debugManager.el.querySelector("button.o_dropdown_toggler"));
+    assert.containsOnce(debugManager.el, "ul.o_dropdown_menu li.o_dropdown_item");
+    const item = debugManager.el.querySelector("ul.o_dropdown_menu li.o_dropdown_item span");
+    assert.strictEqual(item.textContent, "Regenerate Assets Bundles");
+    await click(item);
+    assert.verifySteps(["ir.attachment/search", "ir.attachment/unlink", "reloadPage"]);
+    debugManager.destroy();
```

Comment:
I think we now encourage the use of `registerCleanup` to ensure that the clean up is done even if the test crashes (so basically, juste below line 215)